.github/workflows/extension_ci.yml

```diff
@@ -151,25 +152,24 @@ jobs:
           trunk publish pgmq --version ${pgmq_ver} --description "A lightweight message queue. Like AWS SQS and RSMQ, on Postgres." --homepage "https://github.com/tembo-io/pgmq" --repository "https://github.com/tembo-io/pgmq" --license "PostgreSQL" --category featured --category orchestration
   build_and_push:
     name: Build and push images
-    # needs:
-    #   - publish
-    runs-on:
-      - self-hosted
-      - dind
-      - large-8x8
```

Comment:
pgrx can take quite a bit of resources. I think we'll want to use the self hosted runners for this.

Reply:
once we are completely moved off pgrx then this should no longer be a concern

Reply:
I am not familiar with your build system yet so some times I miss configurations.